include/rocksdb/options.h

```diff
@@ -2080,4 +2080,10 @@ struct LiveFilesStorageInfoOptions {
   uint64_t wal_size_for_flush = 0;
 };
 
+struct WaitForCompactOptions {
+  // If true and PauseBackgroundWork() is called while waiting, Status::Aborted
+  // is returned immediately
```

Comment:
Technically, this leaves unspecified the handling of PauseBackgroundWork() when set to false. (In a case like this, it might not be totally clear whether the opposite of a "must" behavior is "may" or "won't".) Also, what if PauseBackgroundWork() is outstanding before WaitForCompact() is called?

Reply:
>what if PauseBackgroundWork() is outstanding before WaitForCompact() is called

`Status::Aborted` will be returned as well. I will update the comment to include this as well as the default false case.